<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>634</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EAR Audio Rectifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Pics.qrc">
    <normaloff>:/Logo.png</normaloff>:/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveLeft">
   <property name="text">
    <string>Left -&gt; File</string>
   </property>
  </action>
  <action name="actionSaveRight">
   <property name="text">
    <string>Right -&gt; File</string>
   </property>
  </action>
  <action name="actionLoadLeft">
   <property name="text">
    <string>File -&gt; Left</string>
   </property>
  </action>
  <action name="actionLoadRight">
   <property name="text">
    <string>File -&gt; Right</string>
   </property>
  </action>
  <action name="calibrateAction">
   <property name="text">
    <string>Calibrate Latency</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Pics.qrc"/>
 </resources>
 <connections/>
</ui>
